File: chatwoot/ruby/src/GetAccountConversationMetricsExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
    # config.api_key["api_access_token"] = "AGENT_BOT_API_KEY"
    # config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

begin
    response = ChatwootClient::ReportsApi.new.get_account_conversation_metrics(
        0, # account_id
        "account", # type
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling ReportsApi#get_account_conversation_metrics: #{e}"
end

File: chatwoot/typescript-axios/src/GetAccountConversationMetricsExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
  // apiKey: "AGENT_BOT_API_KEY",
  // apiKey: "PLATFORM_APP_API_KEY",
});

new api.ReportsApi(configuration).getAccountConversationMetrics(
  0, // accountId
  "account", // type
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ReportsApi#getAccountConversationMetrics:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
